src/sentry/middleware/ratelimit.py

```diff
@@ -91,6 +92,9 @@ def process_view(self, request: Request, view_func, view_args, view_kwargs) -> R
                     request.will_be_rate_limited = True
                     enforce_rate_limit = getattr(view_class, "enforce_rate_limit", False)
                     if enforce_rate_limit:
+                        logger.exception(
+                            f"Rate limit exceeded for key `{request.rate_limit_key}` hitting {request.build_absolute_uri()}"
```

Comment:
Are these the correct things to be logging?

Reply:
I think this is gonna create an issue every time someone exceed the rate limit. let's just do `logger.info`